pyproject.toml

```
@@ -26,7 +26,7 @@ dependencies = [
   "aiohttp==3.9.5",
   "aiohttp-cors==0.7",
   "aiohttp-jinja2==1.5",
-  "aioipfs @ git+https://github.com/aleph-im/aioipfs.git@d671c79b2871bb4d6c8877ba1e7f3ffbe7d20b71",
+  "aioipfs>=0.7.1",
```

you wan't to cap the maximum version to avoid random breakage

---

```suggestion
  "aioipfs~=0.7.1",
```
Yes, I agree, we can link it to a patch version to avoid future breaks